src/rt/mod.rs

```diff
@@ -0,0 +1,16 @@
+//! Runtime components
+//!
+//! By default, hyper includes the [tokio](https://tokio.rs) runtime.
+//!
+//! If the `runtime` feature is disabled, the types in this module can be used
+//! to plug in other runtimes.
+
+pub mod timer;
+
+pub use timer::*;
```

Comment:
I'd suggest this be an explicit list, just to prevent any accidents now or in the future.

Reply:
[2e9fdb6](https://github.com/hyperium/hyper/pull/3125/commits/2e9fdb69975c15bb8ab70e0a7e70cd275b78155f)